guides/testing.md

```diff
@@ -20,60 +44,92 @@ Concrete test cases are defined in `it` blocks. An optional (but strongly recomm
 
 Test cases that have been defined or outlined but are not yet expected to work can be defined using `pending` instead of `it`. They will not be run but show up in the spec report as pending.
 
-Specs then use the `should` or `should_not` method on actual to verify that the value matches the expectation provided as argument.
+An `it` block contains an example that should invoke the code to be tested and define what is expected of it.
 
-### Expectations
+When `spec` is included, every object has the instance methods `#should` and `#should_not`. These methods are invoked on the value being tested with an expectation as argument. If the expectation is met, code execution continues. Otherwise the example has *failed* and other code in this block will not be executed.
+
+Each example can contain multiple expectations, but it should test only one specific behaviour.
+
+## Expectations
 
 Expectations define if the value being tested (*actual*) matches a certain value or specific criteria.
 
-| Matcher        | Description |
-| --             | -- |
-| `eq(value)`    | passes if actual equals *value* (`==`) |
-| `be(value)`    | passes if actual and *value* are identical (`.same?`) |
-| `be_nil`       | passes if actual is nil (`== nil`) |
-| `be_true`      | passes if actual is true (`== true`) |
-| `be_false`     | passes if actual is false (`== false`) |
-| `be_truthy`    | passes if actual is truthy (neither `nil` nor `false`) |
-| `be_falsey`    | passes if actual is falsy (`nil` or `false`) |
-| `be < value`  | passes if actual is lesser than *value* (`<`) |
-| `be <= value` | passes if actual is lesser than or equal *value* (`<=`) |
-| `be > value`  | passes if actual is greater than *value* (`>`) |
-| `be >= value` | passes if actual is greater than or equal *value* (`>=`) |
-| `be_close(expected, delta)` | passes if actual is within *delta* of *expected* |
-| `contain`      | passes if actual includes *expected* (`.includes?`) |
-| `match(value)` | passes if actual matches *value* (`=~`) |
-| `be_a(type)`   | passes if actual is of type *type* (`is_a?`) |
-| `expect_raises(&block)` | runs the block and passes if it raises an exception |
-| `expect_raises(klass, &block)` | runs the block and passes if it raises an exception of type *klass* |
-| `expect_raises(klass, message, &block)` | runs the block and passes if it raises an exception of type *klass* and the error message contains *message* |
+### Equivalence, Identity and Type
+```crystal
+actual.should eq(expected)    # passes if actual == expected
+actual.should be(expected)    # passes if actual.same?(expected)
+actual.should be_a(expected)  # passes if actual.is_a?(expected)
+actual.should be_nil          # passes if actual.nil?
+```
+
+### Truthiness
```

Comment:
As this section is not _only_ about truthiness, maybe we could named it `Thruthiness, Falseiness` (not sure it should be spelled though)

Reply:
I'm sure *truthiness* can be used as generic term for both ([example](https://learn.co/lessons/truthiness-in-ruby-readme))